tools/ci_generate.nim

```diff
@@ -140,6 +185,7 @@ packages:
   writeFile("build_all.sh", genPosixScript())
   writeFile("build_all.bat", genWindowsScript(buildAll = true))
   writeFile("ci/build_autogen.bat", genWindowsScript(buildAll = false))
+  writeFile("drone.yml", genDroneScript())
```

Comment:
shouldn't that be .drone.yml? (IIRC that's what's commonly used, and the default)

I've enabled drone for nim-lang/nim, see https://cloud.drone.io/nim-lang/Nim/settings
(and assumes .drone.yml, was was the default, although we could configure it but not sure it'd work for forks unless the fork also sets this?)


![image](https://user-images.githubusercontent.com/2194784/118737154-190db300-b7f9-11eb-979e-38bd52fb9175.png)